core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/BulkImport.java

```diff
@@ -449,31 +450,31 @@ private Text toText(byte[] row) {
 
   }
 
-  private static List<FileStatus> filterInvalid(FileStatus[] files) {
-    ArrayList<FileStatus> fileList = new ArrayList<>(files.length);
+  public static List<FileStatus> filterInvalid(FileStatus[] files) {
 
+    ArrayList<FileStatus> fileList = new ArrayList<>(files.length);
     for (FileStatus fileStatus : files) {
 
       String fname = fileStatus.getPath().getName();
 
+      if (fileStatus.isDirectory()) {
+        log.debug("{} is a directory, ignoring.", fileStatus.getPath());
+        continue;
+      }
+
       if (fname.equals("_SUCCESS") || fname.equals("_logs")) {
```

Comment:
It would be good to add these files to the list of working files.

Reply:
Agree, they are hadoop working files that occur during the Bulk processing. Also, found a hadoop constant ...FileOutputCommitter.SUCCEEDED but can not locate a constant for "_logs".